dih_models/parameters.py

```
@@ -197,7 +199,7 @@ class Parameter(float):
     __slots__ = (
         'manual_ref', 'source_ref', 'source_type', 'description', 'unit', 'formula', 'latex',
         'confidence', 'last_updated', 'peer_reviewed', 'conservative',
-        'sensitivity', 'display_value', 'display_name', 'keywords',
+        'sensitivity', 'display_value', 'display_name', 'chart_label', 'interval_label', 'keywords',
         'validation_min', 'validation_max', 'confidence_interval', 'std_error',
         'distribution', 'inputs', 'compute', 'latex_symbol', 'hide_ci'
     )
```

Ruff (0.16.2)

[warning] `Parameter.__slots__` is not sorted

Apply a natural sort to `Parameter.__slots__`

(RUF023)